src/components/charts/cardLineChart.tsx

```diff
@@ -68,7 +68,7 @@ const CardLineChart = ({ ratingData, displayLegend = false, className }: ChartLi
         const chart = new Chart(ctx!, getConfig(datasets, displayLegend));
 
         return () => chart.destroy();
-    }, [chartContainer, ratingData]);
```

Comment:
Please verify that the chart refreshes correctly in "hot reload" mode during development. I remember that I had issues with this but not sure if putting container inside deps was the solution.

Reply:
I don't see any problems with "hot reload" without chartContainer in deps